packages/grafana-data/src/types/live.ts

```diff
@@ -149,6 +150,34 @@ export interface LiveChannelAddress {
   path: string;
 }
 
+/**
+ * Return an address from a string
+ *
+ * @alpha -- experimental
+ */
+export function parseLiveChannelAddress(id: string): LiveChannelAddress | undefined {
+  if (!id || id.length) {
+    return undefined;
+  }
+  const idx0 = id.indexOf('/');
+  if (idx0 < 1) {
+    return undefined;
+  }
+  const idx1 = id.indexOf('/', idx0 + 1);
+  if (idx1 < 1) {
+    return undefined;
+  }
+  const idx2 = id.indexOf('/', idx1 + 1);
+  if (idx2 < 1) {
+    return undefined;
+  }
+  return {
+    scope: id.substring(0, idx0) as LiveChannelScope,
+    namespace: id.substring(idx0 + 1, idx1),
+    path: id.substring(idx2),
+  };
+}
```

Comment:
is this an ultra fancy way of doing this?

```suggestion
export function parseLiveChannelAddress(id: string): LiveChannelAddress | undefined {
    let parts = id.split("/");

    if (parts.length === 3) {
      return {
          scope:     parts[0] as LiveChannelScope,
          namespace: parts[1],
          path:      parts[2],
      };   
    }
}
```

Reply:
:grin: probably -- except path can have more slashes...  but ya